easybuild/tools/options.py

```diff
@@ -1432,7 +1433,8 @@ def set_up_configuration(args=None, logfile=None, testing=False, silent=False):
         if eb_go.options.include_easyblocks:
             # make sure we're not including the same easyblock twice
             included_from_pr = set([os.path.basename(eb) for eb in easyblocks_from_pr])
-            included_from_file = set([os.path.basename(eb) for eb in eb_go.options.include_easyblocks])
+            included_paths = expand_glob_paths(eb_go.options.include_easyblocks)
+            included_from_file = set([os.path.basename(eb) for eb in included_paths])
             included_twice = included_from_pr & included_from_file
             if included_twice:
                 raise EasyBuildError("Multiple inclusion of %s, check your --include-easyblocks options",
```

Comment:
@migueldiascosta Rather than raising an error, I think we should let the easyblocks that are included from the PR win over any local easyblocks included with `--include-easyblocks`. That makes perfect sense to me, since I don't see a use case where you would be using `--include-easyblocks-from-pr` but you would still like to see local easyblocks get precedence?

It's probably also a good idea to print a message for the easyblocks that get pulled in from a PR, to make that a bit more visible?

```
== easyblock example.py included from PR #1234
```
```

Reply:
Actually, the easyblocks included from the PR should already take precedence, because they are included later and their path is prepended. 

I hadn't realised this before but when the local easyblocks are included, they are immediately imported in `verify_imports` and they remain in `sys.modules`. When the same easyblock is included later from a PR, `verify_imports` fails because it continues to use previously imported module.

Not sure if this would be the best approach, but at least to demonstrate what is happening, if we add `sys.modules.pop(pymod_spec)` at the end of the loop in `verify_imports`, then the error reported by @zao goes away, the easyblocks from a PR take precedence, and we can stop raising this error